reventlov/plugins/__init__.py

```diff
@@ -13,18 +13,27 @@ def iter_namespaces():
     )
 
 
-plugins = {
-    name: importlib.import_module(name)
-    for finder, name, ispkg
-    in iter_namespaces()
-}
+def find_plugins():
+    excluded = (os.getenv('REVENTLOV_DISABLED_PLUGINS') or '').split(',')
+    plugins = {
+        name.split('.')[-1]: importlib.import_module(name)
+        for finder, name, ispkg
+        in iter_namespaces()
+        if name.split('.')[-1] not in excluded
+    }
```

Comment:
Not sure if the indentation is PEP8 compliant here...

Reply:
Well, `pycodestyle` is not complaining on this.